src/lib/format.ts

```diff
@@ -23,63 +17,3 @@ export const toTypesPillar = (p: Pillar): TypesPillar => {
             return TypesPillar.News;
     }
 };
-
-const toTypesDisplay = (d: Display): TypesDisplay => {
-    switch (d) {
-        case Display.Standard:
-            return TypesDisplay.Standard;
-        case Display.Immersive:
-            return TypesDisplay.Immersive;
-        case Display.Showcase:
-            return TypesDisplay.Showcase;
-        default:
-            return TypesDisplay.Standard;
-    }
-};
-
-const toTypesDesign = (d: DesignType): TypesDesign => {
```

Comment:
This function wasn't used and is no longer needed now that we import directly from /types